gnovm/pkg/gnolang/uverse.go

```diff
@@ -848,6 +856,17 @@ func makeUverseNode() {
 		),
 		nil, // results
 		func(m *Machine) {
+			if bm.NativeEnabled {
+				arg0 := m.LastBlock().GetParams1(m.Store)
+				bm.StartNative(bm.GetNativePrintCode(uverseLen(m, arg0, false)))
+				prevOutput := m.Output
+				m.Output = os.Stdout
```

Comment:
I’m not sure this is the right benchmark, since in production GnoVM m.Output is nil, not stdout. This will cause significant false time consumption.